bvm/ballerina-core/src/main/java/org/ballerinalang/bre/coverage/ExecutedInstruction.java

```diff
@@ -0,0 +1,148 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.bre.coverage;
+
+import org.ballerinalang.util.codegen.LineNumberInfo;
+
+/**
+ * This is the data holder for executed instruction which is used for coverage data.
+ *
+ * @since 0.985
+ */
+public class ExecutedInstruction {
+
+    private int ip;
+
```

Comment:
Shall we remove these additional blank lines between variable definitions?

Reply:
Usually blank lines are kept for readability. I think class def and instance variables are 2 different groups and they should be logically grouped. Thus I think here empty lines are good

Reply:
Yeah thats true, I am suggesting to remove blank lines **between variable definitions** :)

Reply:
Ah, WSO2 specific standards ;)